wiki/osu!_wiki/Maintenance/en.md

```diff
@@ -0,0 +1,104 @@
+
+The CI checks are run automatically on every commit of a recurring contributor. In order to have their pull requests merged, contributors are expected to fix errors reported by the CI. To see the [status of checks](img/ci-status.png), do the following:
+
+1. Scroll the pull request's page down, find the `osu-wiki continuous integration` status row, and click the `Details` link.
+2. On the new page, expand the `run remark on changed files` step. Each finding is accompanied by a its exact location in a file and a short description of why is it an error.
+
+If you need help with decrypting the CI check errors, or fixing the issues, ask in the `#osu-wiki` channel on Discord.
+
+### Development
+
+osu! wiki is integrated into the osu! website, which means all technical feature requests should be made and tracked in the [`ppy/osu-web` repository](https://github.com/ppy/osu-web/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aarea%3Awiki). To inform other contributors of your request, make sure to link the new issue in the `#osu-wiki` Discord channel, or osu! wiki's issue tracker.
+
+### Tools
+
+Some features are not directly related to the osu! website, but may be useful for contributing or general housekeeping. In this case, they are implemented by people capable of it and are **not** added to the website directly:
+
+- [osu-wiki status](https://clayton.cc/osu-wiki-status/en): list of articles per every language, and the category of maintenance they require (translation, update, stub expansion). Functionality requests: [ppy/osu-wiki#2486](https://github.com/ppy/osu-wiki/issues/2486)
+- [osu-wiki-bin](https://github.com/cl8n/osu-wiki-bin): a Node.js utility for automated checks and edits (broken links, usergroup updates, wiki-wide text replacements, and other things)
+- [flag-wiki-osu](https://megaapplepi.github.io/flag-wiki-osu/): automatically add references to the flags mentioned in an article. Functionality requests: [ppy/osu-wiki#328](https://github.com/ppy/osu-wiki/issues/328)
+
+## Routines
+
+*Note: the [osu-wiki status](https://clayton.cc/osu-wiki-status/en) website shows list of all articles in need of maintenance, broken down by category.*
+
+The wiki relies on input from the osu! community. You can help the maintainers and other contributors by doing your part. For information on how to do that, read the [contribution guide](/wiki/osu!_wiki_Contribution_Guide) If at any point you feel stuck, ask for help in the `#osu-wiki` channel of [osu!dev Discord server](/wiki/osu!dev_Discord_server).
+
+### Translations
+
+<!-- note: the GitHub links are intentional here, because they expose many articles of a category at once -->
+
+*For the list of translations and their completeness, see [osu-wiki status](https://clayton.cc/osu-wiki-status/en).*
```

Comment:
https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#for-see (grave marks are a typo)

This ASC formatting keeps getting forgotten (maybe it's time to change it?).

Reply:
you know, I never remember the exact form. dunno what to make of it